schemas/test/en/test_list_collector_one_related_answer.json

```diff
@@ -0,0 +1,312 @@
+{
```

Comment:
Do we even need a schema?. It can just be unit tested?. We've been trying to reduce no. of schemas, is this really needed?.

Reply:
OK removed the test_schema now and just kept the summary item config unit test 👍